x-pack/plugins/security_solution/common/detection_engine/schemas/common/schemas.ts

```diff
@@ -188,6 +188,17 @@ export type AnomalyThreshold = t.TypeOf<typeof PositiveInteger>;
 export const anomalyThresholdOrUndefined = t.union([anomaly_threshold, t.undefined]);
 export type AnomalyThresholdOrUndefined = t.TypeOf<typeof anomalyThresholdOrUndefined>;
 
+export const should_disable_timestamp_fallback = t.boolean;
```

Comment:
Do you think a more concise name like `disable_timestamp_fallback` would be ok?

Reply:
done